firestore-bigquery-export/scripts/import/package.json

```diff
@@ -25,6 +25,7 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@firebaseextensions/firestore-bigquery-change-tracker": "^1.1.2",
+    "commander": "5.0.0",
```

Comment:
Can we change & test this against `^5.1.0`?

Reply:
![image](https://user-images.githubusercontent.com/1075793/86911446-3c64cd80-c113-11ea-9f3d-e7c1ba40a38f.png)
It worked after updating to `^5.1.0`
Do you want me to push the change? I see you resolved this conversation so I'm not going to without your go ahead.